src/styles/global.scss

```diff
@@ -11,6 +11,7 @@
   color: #3d3935;
   --main-color: #237a57;
   --sub-color: #f2f0eac8;
+  --transition: all 400ms ease;
```

Comment:
It's okay for now, but if you are using sass, please use variables and put them in styles/_variables.scss.

Reply:
I added common style to _variables